content/docs/1.1.0/volumes-and-nodes/disks-or-nodes-eviction.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Disks or Nodes Eviction Support
+weight: 5
+---
+
+Longhorn supports auto eviction for evicting the replicas on the selected disabled disks or nodes to other suitable disks and nodes. Meanwhile keep the same level of fault tolerance during this eviction period of time.
+
+> **Note:** This eviction feature can only be enabled when the selected disks or nodes are scheduling disabled. And during the eviction time, the selected disks or nodes can not be re-enabled for scheduling.
+
+> **Note:** This eviction feature is only valid for `Healthy` volumes, which means `Attached` and `Healthy`. If the volume is `Detached`, the eviction won't work for this volume.
```

Comment:
It should work on the degraded volume as well?

Reply:
I am wondering if it's in degraded state, that means longhorn either can not find new disk for new replica or something is wrong with replica rebuild, right?

Reply:
It can be it's rebuilding a replica too.

Reply:
yes, you are right.